partner_restrict_payment_acquirer/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to restrict payment acquirers for partner. Only selected acquirers will be available on invoice payment page.
```

Comment:
Please modify the following part
`Only selected acquirers will be available for online payment in portal.`